src/mocks/data.ts

```diff
@@ -1,5 +1,73 @@
 import { FetchResponse } from "./../common/types";
-import { DeviceType, ProjectType, RecordType } from "./../common/interfaces";
+import {
+  DateValueType,
+  DeviceType,
+  ProjectType,
+  RecordType,
+} from "./../common/interfaces";
+
+const getFakeLineChartDates = (): DateValueType[] => [
+  {
+    date: "2021-04-11T12:10:01.908Z",
+    value: 25,
+  },
+  {
+    date: "2021-04-10T12:10:01.908Z",
+    value: 99,
+  },
+  {
+    date: "2021-04-09T12:10:01.908Z",
+    value: 75,
+  },
+  {
+    date: "2021-04-08T12:10:01.908Z",
+    value: 12,
+  },
+  {
+    date: "2021-04-07T12:10:01.908Z",
+    value: 0,
+  },
+  {
+    date: "2021-04-06T12:10:01.908Z",
+    value: 33,
+  },
+];
+
+export const fakeCuratedProjects = [
```

Comment:
This could be a subset of `./supabaseData/publicProjects.ts` to make it more realistic?

Reply:
Good point! Will do!

Reply:
See 3539a02